src/pages/RegisterPage.vue

```diff
@@ -10,14 +10,16 @@
 </template>
 
 <script>
-import MainTitle from '../components/MainTitle.vue';
-import FormRegister from '../components/FormRegister.vue';
-
+import MainTitle from "../components/MainTitle.vue";
+import FormRegister from "../components/FormRegister.vue";
 
 export default {
-    name: 'RegisterPage',
-    components: {MainTitle, FormRegister},
-}
+  name: "RegisterPage",
+  metaInfo: {
+    title: "Register Page Nr 1",
```

Comment:
There is nothing wrong with this name but try to imagine when it appears on the browser tab. Maybe you will find out some more elegant way or maybe just leave same name for both steps or leave it this way.